src/controllers/auth.controller.js

```diff
@@ -100,7 +112,11 @@ let authController = (authService, userService, templateModel) => {
         })
       }
     }).then(user => {
-      return resetPasswordEmail(user)
+      // Reset Password Email
+      let params = []
```

Comment:
Hi @kevinpr007,

If it's empty, then there's no use to send it as a parameter. If we need to send parameters later on, we'll do a refactor.